src/main/java/com/aro/javaadmin/student/StudentDTO.java

```diff
@@ -0,0 +1,21 @@
+package com.aro.javaadmin.student;
+
+import com.aro.javaadmin.user.User;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class StudentDTO {
+
+
+    private Long studentId;
+
+    private String firstName;
+
+    private String lastName;
+
+    private String level;
+
+    private User user;
```

Comment:
Entity should not be a part of DTO. If you want to keep this design use existing UserDTO. Second thing is that any passwords once put into a system should never leave it at any circumstances. They are only collected to be processed internally and should never be exposed - no matter if it's a secured API or not.